modin/experimental/engines/omnisci_on_ray/frame/data.py

```diff
@@ -555,16 +555,16 @@ def join(self, other, how="inner", on=None, sort=False, suffixes=("_x", "_y")):
 
         conflicting_cols = set(self.columns) & set(other.columns) - set(on)
         for c in self.columns:
-            suffix = suffixes[0] if c in conflicting_cols else ""
-            new_columns.append(c + suffix)
+            new_name = str(c) + suffixes[0] if c in conflicting_cols else c
```

Comment:
```suggestion
            new_name = f"{c}{suffixes[0]}" if c in conflicting_cols else c
```
or
```suggestion
            new_name = (str(c) + suffixes[0]) if c in conflicting_cols else c
```
to avoid ambiguity of order when reading this `+` and `if..else` operations.

Reply:
Fixed